### PR TITLE
Add custom dispatcher to map-iterator-prototype

### PR DESCRIPTION
**Pull request by orkvi**
JerryScript-DCO-1.0-Signed-off-by: Orkenyi Virag orkvi@inf.u-szeged.hu

### CODE REVIEW

File: jerry-core/ecma/builtin-objects/ecma-builtin-map-iterator-prototype.c [hunk 1]
```diff
@@ -54,6 +68,35 @@ ecma_builtin_map_iterator_prototype_object_next (ecma_value_t this_val) /**< thi
...
+ecma_builtin_map_iterator_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide
+                                                                                   *   routine identifier */
+                                     ecma_value_t this_arg, /**< 'this' argument value */
+                                     const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                             *   passed to routine */
+                                     uint32_t arguments_number) /**< length of arguments' list */
+{
+  JERRY_UNUSED (arguments_list_p);
+  JERRY_UNUSED (arguments_number);
+  switch (builtin_routine_id)
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-map-iterator-prototype.c:87]: new line before

File: jerry-core/ecma/builtin-objects/ecma-builtin-map-iterator-prototype.c [hunk 2]
```diff
@@ -54,6 +68,35 @@ ecma_builtin_map_iterator_prototype_object_next (ecma_value_t this_val) /**< thi
...
+ * @return ecma value
+ *         Returned value must be freed with ecma_free_value.
+ */
+ecma_value_t
+ecma_builtin_map_iterator_prototype_dispatch_routine (uint8_t builtin_routine_id, /**< built-in wide
+                                                                                   *   routine identifier */
+                                     ecma_value_t this_arg, /**< 'this' argument value */
+                                     const ecma_value_t arguments_list_p[], /**< list of arguments
+                                                                             *   passed to routine */
+                                     uint32_t arguments_number) /**< length of arguments' list */
```
Comment (rerobika) [inline jerry-core/ecma/builtin-objects/ecma-builtin-map-iterator-prototype.c:83]: misaligned parameters.

**Comment by zherczeg** [comment]
Is there a binary size gain in this case? Or just for consistency?

**Comment by zherczeg** [comment]
Patch looks ok, but I had a question above.

**Comment by orkvi** [comment]
> Is there a binary size gain in this case? Or just for consistency?

Just for consistency.

**Review by galpeter — APPROVED** [review]
lgtm

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**